content/hub/template_http_js.md

```diff
@@ -0,0 +1,22 @@
+title = "JavaScript HTTP trigger template"
+template = "render_hub_content_body"
+date = "2022-10-15T00:22:56Z"
+content-type = "text/html"
+tags = ["javascript", "http"]
+
+[extra]
+author = "fermyon"
+type = "hub_document"
+category = "Template"
+language = "JavaScript"
+created_at = "2022-10-15T00:22:56Z"
+last_updated = "2022-10-15T00:22:56Z"
+spin_version = ">v0.8"
```

Comment:
```suggestion
spin_version = ">v1.4"
```

It is going to be greater than v1.4 once we cut a new release tomorrow with the sqlite stuff. Technically v1.3 but yes

Reply:
No, because if someone is on Spin 0.8 and installs the JS template, they'll get a working app, no?  (Assuming the JS repo is template tagged, that is.)  This is why we version templates... if we adopt this philosophy we'll have to _continually rev every template entry_ to say it requires latest Spin.

Reply:
Oh yes, that actually makes sense.